pkg/transcriptionsystems/bipa/consonants.tsv

```diff
@@ -175,6 +176,7 @@ m	voiced	bilabial	nasal
 mʰ	voiced	bilabial	nasal		aspiration:aspirated	
 mː	voiced	bilabial	nasal		duration:long	
 mʷ	voiced	bilabial	nasal		labialization:labialized	
+mw	voiced	bilabial	nasal	+	labialization:labialized	
```

Comment:
I consider these things as dangerous, as this can also be a double articulation. So I prefer not to use the alias here, but rather to link explicitly in datasets where this is used to express m + superscript w

Reply:
I'll remove it.